source/users/create.txt

```diff
@@ -226,39 +345,67 @@ There is no such specific method to re-run a custom confirmation
-the :doc:`App Services Admin API </admin/api/v3>` or via the App Services UI:
+the the App Services UI or the :doc:`App Services Admin API </admin/api/v3>`:
 
-To re-run the confirmation workflow for a user through the App Services UI:
+.. tabs-realm-admin-interfaces::
+   
+   .. tab::
+      :tabid: ui
 
-#. Select :guilabel:`App Users` from the left navigation menu.
+      To re-run the confirmation workflow for a user in the UI:
 
-#. Under the :guilabel:`Users` tab, select the :guilabel:`PENDING`
-   button.
+      #. Select :guilabel:`App Users` from the left navigation menu.
 
-#. Find the user in the list and click on the ellipsis (``...``).
+      #. Under the :guilabel:`Users` tab, select the :guilabel:`PENDING`
+         button.
 
-#. Select the :guilabel:`Run user confirmation` option from the context
-   menu that appears.
+      #. Find the user in the list and click on the ellipsis (``...``).
 
-#. Select the :guilabel:`Run User Confirmation` button in the dialogue
-   box that appears.
+      #. Select the :guilabel:`Run user confirmation` option from the context
+         menu that appears.
 
-#. If the operation succeeds, the banner at the top of the App Services
-   admin console should display a confirmation message. The user's
-   :guilabel:`User Status` changes from :guilabel:`Pending Confirmation`
-   to :guilabel:`Pending User Login`. Once the user logs in, they will
-   move into the active users list automatically.
+      #. Select the :guilabel:`Run User Confirmation` button in the dialogue
+         box that appears.
 
-If the re-run fails or the user's tokens expire again, you can re-run
-the confirmation function as many times as necessary.
+      #. If the operation succeeds, the banner at the top of the App Services
+         admin console should display a confirmation message. The user's
+         :guilabel:`User Status` changes from :guilabel:`Pending Confirmation`
+         to :guilabel:`Pending User Login`. Once the user logs in, they will
+         move into the active users list automatically.
+
+      If the re-run fails or the user's tokens expire again, you can re-run
+      the confirmation function as many times as necessary.
+
+   .. tab::
+      :tabid: api
+
+      To re-run the confirmation workflow for a user using the Admin API, create a 
+      ``POST`` request in the following format. You must specify the Group ID, App ID, 
+      and email address.
+
+      .. code-block:: sh
+
+         curl --request POST \
+           --header 'Authorization: Bearer <access_token>' \
+           https://realm.mongodb.com/api/admin/v3.0/groups/<groupId>/apps/<appId>/user_registrations/by_email/<email>/run_confirm
+
+      .. seealso::
+       
+         :admin-api-endpoint:`Admin API Documentation <operation/adminRerunPendingUserConfirmation>`
 
 Summary
 -------
 
+- For most providers, App Services automatically creates a user object the first time 
```

Comment:
Can we mention that this isn't for email password users in this summary sentence.